build/taskctl/tasks.yaml

```diff
@@ -1,4 +1,11 @@
 tasks:
+  image-pull:
+    context: powershell
+    command: ""
+
+  image-pull-dotnet:
+    context: powershell-dotnet
+    command: ""
```

Comment:
What are these for?

Reply:
They're just for pulling the image as a separate task so the build log doesn't show the pulling clutter the first time an image is used

Reply:
Ah that makes sense. It may be worth adding a comment then. I know my first instinct would be to remove them without that context

Reply:
Will do, thanks